test/complexityRule.spec.ts

```diff
@@ -0,0 +1,100 @@
+// tslint:disable:max-line-length
+import { assertSuccess, assertAnnotated } from './testHelper';
+import { Replacement } from 'tslint';
+import { expect } from 'chai';
+
+describe('complexity', () => {
+    describe('success', () => {
+        it('should work with a lower level of complexity', () => {
+            let source = `
+      @Component({
+        template: \`
+        <div *ngIf="a === '1' || (b === '2' && c.d !== e)">
```

Comment:
This will fail to parse if the expression is:

```
a === '1' || b === '2' && c.d !== e
```
right?

Reply:
No. I added a test with this expression.